docs/DeveloperGuide.md

```diff
@@ -149,107 +149,56 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Implementation**
+#### Implementation
 
-This section describes some noteworthy details on how certain features are implemented.
+This section describes some noteworthy feature implementation details of KnottyPlanners.
 
-### \[Proposed\] Undo/redo feature
+### Wedding
 
-#### Proposed Implementation
+The `Wedding` class represents a wedding event in KnottyPlanners. It includes details such as the wedding name, venue,
+ date and participants. The `Wedding` class supports the following functions:
 
-The proposed undo/redo mechanism is facilitated by `VersionedAddressBook`. It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+* `Add Wedding`: Creates a new `Wedding` in KnottyPlanners.
+* `Delete Wedding`: Deletes an existing `Wedding` from KnottyPlanners and the associated tags.
+* `List Weddings`: Lists all `Weddings` in KnottyPlanners.
+* `View Wedding`: Displays participants of a specific `Wedding`.
 
-* `VersionedAddressBook#commit()` — Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` — Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` — Restores a previously undone address book state from its history.
+### StaticContext for Deletion and Clear
```

Comment:
maybe can be clearer with the title, so something like `Confirmation prompts for delete person, delete wedding and clear commands`

Reply:
roger